### DISCUSSION TITLE
Why always there is an empty array of Synonyms and Antonyms?

### DISCUSSION THREAD

**Discussion by almokhtarbr** [meetDeveloper/freeDictionaryAPI#139] [Q&A]
Every word tried to get meaning for getting an empty array of Synonyms and Antonyms? is there something am I missing?

**Comment by JWilkie1028** [comment]
The inner definition synonyms and antonyms are empty, but the ones outside the definition sometimes have content.
Note there are no entries in the meaning, but entries after the meanings - dunno. Maybe that's just the evolution of the Json layout?

"meanings": [
    {
      "partOfSpeech": "verb",
      "definitions": [
        {
          "definition": "To disclaim and expel from the family, as a father his child; to disown; to disinherit.",
          "synonyms": [],
          "antonyms": []
        },
        {
          "definition": "To formally separate oneself from or to divest oneself of.",
          "synonyms": [],
          "antonyms": []
        },
        {
          "definition": "To depose.",
          "synonyms": [],
          "antonyms": []
        },
        {
          "definition": "To reject; to cast off; to discard.",
          "synonyms": [],
          "antonyms": []
        },
        {
          "definition": "To surrender, renounce or relinquish, as sovereign power; to withdraw definitely from filling or exercising, as a high office, station, dignity; to fail to fulfill responsibility for.",
          "synonyms": [],
          "antonyms": [],
          "example": "Note: The word abdicate was held to mean, in the case of James II, to abandon without a formal surrender."
        },
        {
          "definition": "To relinquish or renounce a throne, or other high office or dignity; to renounce sovereignty.",
          "synonyms": [],
          "antonyms": []
        }
      ],
      "synonyms": [
        "abandon",
        "abjure",
        "cast aside",
        "cast off",
        "cede",
        "desert",
        "disinherit",
        "disown",
        "forego",
        "forsake",
        "give up",
        "quit",
        "refuse",
        "reject",
        "relent",
        "relinquish",
        "renounce",
        "repudiate",
        "resign",
        "retire",
        "stand down",
        "surrender",
        "vacate",
        "waive",
        "yield"
      ],
      "antonyms": [
        "claim",
        "grasp",
        "maintain",
        "occupy",
        "retain",
        "seize",
        "usurp"
      ]
    }
